tool/mustachio/codegen_runtime_renderer.dart

```diff
@@ -316,6 +341,26 @@ class ${renderer._rendererClassName}${renderer._typeParametersString}
         _buffer.writeln('    ...$superMapName$generics(),');
       }
     }
+    if (contextClass.mixins != null) {
+      // Mixins are spread into the property map _after_ the super class, so
+      // that they override any values which need to be overridden. Superclass
+      // and mixins override from left to right, as do spreads:
```

Comment:
is this not right to left?  (N overrides M, which overrides E). Or am I thinking about this wrongly.

Reply:
Hmm yeah maybe the semantics are meaningless. I mean that mixins are _applied_ left to right. So M is applied, then N is applied, overriding things from M.

Reply:
ah yes.  They are applied left to right, which implies that they override right to left.   Implied applies applies implying....

Reply:
😱 😱 😱